### PR TITLE
B and C named icons

### PR DESCRIPTION
**Pull request by nvmusoke**
still going through adding icons. Finished everything starting with a B and C.

### CODE REVIEW

File: src/components/Icon/icons/ChartBars2.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconColorsMap from './colors';
+
+const ChartBars2 = ({ fill, ...props }) => (
```
Comment (chelshaw) [inline src/components/Icon/icons/ChartBars2.js]: Wondering if we should just call it `ChartBars` if we're not also importing the presumed original `ChartBars` ?

Comment (chelshaw) [reply src/components/Icon/icons/ChartBars2.js]: Or maybe name it descriptively like the others

Comment (nvmusoke) [reply src/components/Icon/icons/ChartBars2.js]: I was just going off the names given to me in the svg file, but i agree that a few of them could have better names and happy to change any.

Comment (pixelbandito) [reply src/components/Icon/icons/ChartBars2.js]: I second @chelshaw ! Otherwise looks good.

**Comment by pixelbandito** [comment]
FYI: This should be a minor version bump. I think it's fine to do the version bump in this PR, then wait until it's merged before `yarn publish`ing from master.

**Pull request merged**